sycl/doc/EnvironmentVariables.md

```diff
@@ -238,29 +238,31 @@ variables in production code.</span>
 :warning: **Warning:** <span style="color:red">the environment variables
 described below are used for development and debugging of DPC++ compiler
 and runtime. Their semantics are subject to change. Do not rely on these
-variables in production code.</span>
+variables in production code. Please prefix them with `UR_L0_` or
+`SYCL_PI_LEVEL_ZERO_`. `SYCL_PI_LEVEL_ZERO_` will be deprecated in upcoming
+versions releases. </span>
```

Comment:
@smaslov-intel : Do you think this is ok, or would you prefer having the table below with each entry with the old and new names, something like:

<html>
<body>
<!--StartFragment-->

Environment variable | Values | Description
-- | -- | --
UR_L0_SINGLE_THREAD_MODE, SYCL_PI_LEVEL_ZERO_SINGLE_THREAD_MODE (deprecated) | Integer | A single-threaded app has an opportunity to enable this mode to avoid overhead from mutex locking in the Level Zero plugin. A value greater than 0 enables single thread mode. A value of 0 disables single thread mode. The default is 0.

<!--EndFragment-->
</body>
</html>

Reply:
I think the documentation of UR_L0 env vars should be moved under https://github.com/intel/llvm/tree/sycl/sycl/plugins/unified_runtime/ur/adapters/level_zero

In this file we'd provide a mapping from SYCL_PI_LEVEL_ZERO to UR_L0 ones, and link to their definitions (and also mention that SYCL_PI_LEVEL_ZERO are deprecated).

Reply:
Thanks @smaslov-intel . Moving this to draft then while I open a PR in the adapter and then will update back here once we have a link.